diff --git a/dune b/dune
index 183e0bc..dd3dfe0 100644
--- a/dune
+++ b/dune
@@ -40,7 +40,7 @@
      -G
      "Unix Makefiles"
      ; GCC 7 (shipped with esy) doesn't like _ for unused variables
-     "-DCMAKE_CXX_FLAGS=-Wno-unused-variable -Wno-maybe-uninitialized"
+     "-DCMAKE_CXX_FLAGS=-Wno-unused-variable -Wno-maybe-uninitialized -Wno-stringop-overread"
      -DBUILD_TESTS=OFF
      -DBUILD_TOOLS=OFF
      -DBUILD_STATIC_LIB=ON
@@ -129,7 +129,7 @@
      -DCMAKE_CXX_COMPILER=x86_64-w64-mingw32-g++
      "-DCMAKE_SYSTEM_NAME=Windows"
      ; GCC 7 (shipped with esy) doesn't like _ for unused variables
-     "-DCMAKE_CXX_FLAGS=-Wno-unused-variable -Wno-maybe-uninitialized"
+     "-DCMAKE_CXX_FLAGS=-Wno-unused-variable -Wno-maybe-uninitialized  -Wno-stringop-overread"
      -DBUILD_TESTS=OFF
      -DBUILD_TOOLS=OFF
      -DCMAKE_SHARED_LIBRARY_PREFIX_CXX=lib
